dips/dip-14.md

```diff
@@ -0,0 +1,244 @@
+---
+dip: 14
+title: Multi-Agent Transactions
+author: Emma Zhong(@emmazzz), Tim Zakian, Sam Blackshear
```

Comment:
author -> authors. (otherwise, build fails)

Reply:
Ah that explains a lot.. Thanks!